js/src/ui/TxList/txList.js

```diff
@@ -35,19 +35,16 @@ class TxList extends Component {
       PropTypes.array,
       PropTypes.object
     ]).isRequired,
-    netVersion: PropTypes.string.isRequired
+    blockNumber: PropTypes.object,
+    netVersion: PropTypes.string.isRequired,
+    newError: PropTypes.func
   };
 
-  store = new Store(this.context.api);
-
   componentWillMount () {
+    this.store = new Store(this.context.api, this.props.newError);
```

Comment:
I think you should also cater for `newError` being changed during the lifetime of the component (or when component will be re-used with different props). Isn't it possible to avoid passing the error directly?
Like:
```js
store: new Store(this.context.api, (...args) => this.props.newError(...args))
// or
//store: new Store(this.context.api, (...args) => this.props.newError ? this.props.newError(...args) : null)
```